domain/schema/controller.go

```diff
@@ -252,12 +252,12 @@ CREATE TABLE cloud_region_defaults (
 );
 
 CREATE TABLE cloud_ca_cert (
-    uuid              TEXT PRIMARY KEY,
     cloud_uuid        TEXT NOT NULL,
     ca_cert           TEXT NOT NULL,
     CONSTRAINT        fk_cloud_ca_cert_cloud
         FOREIGN KEY       (cloud_uuid)
         REFERENCES        cloud(uuid)
+    PRIMARY KEY (cloud_uuid, ca_cert)
```

Comment:
Should the primary key not be the cloud_uuid, Traditionally if you have multiple CA's for a cloud you would make one big pem bundle for them.

Reply:
as discussed online, we'll keep it like this